libs/cardano-ledger-api/cardano-ledger-api.cabal

```
@@ -0,0 +1,52 @@
+cabal-version:      2.4
+name:               cardano-ledger-api
+version:            0.1.0.0
+synopsis:           Public API for the cardano ledger codebase
+description:
+  This package a public interface into the cardano ledger codebase. It also
+  provides functionality that works for all eras and is useful for downstream
+  packages.
+bug-reports:        https://github.com/input-output-hk/cardano-ledger/issues
+
+license:            Apache-2.0
+author:             IOHK Formal Methods Team
```

```suggestion
author:             IOG Ledger Team
```

Using "Formal Methods" is probably left over from when this was the ledger specs repo. I _think_ IOG Ledger Team is better.

---

This was copy and paster. I've created a ticket for that: https://github.com/input-output-hk/cardano-ledger/issues/2999

All cabal files suffer from this, so it is best to take care of this as one task. So, if you can comment on that ticket what the proper values should be that will be great.

---

perfect, thanks!